docs/essentials/exceptions/exceptions-best-practices.md

```diff
@@ -192,29 +97,15 @@ The following list contains guidelines for creating your own exceptions and when
   }
   ```
 
-  C++
-  ```
-  public ref class MyFileNotFoundException : public Exception
-  {
-  };
-  ```
-
-  Visual Basic
-  ```
-  Public Class MyFileNotFoundException
-      Inherits Exception
-  End Class
-  ```
-
 - In C# and C++, use at least the three common constructors when creating your own exception classes: the default constructor, a constructor that takes a string message, and a constructor that takes a string message and an inner exception. For an example, see How to: Create User-Defined Exceptions.
```

Comment:
As I was lead to believe, only C# is currently supported in .NET Core. If this is incorrect, though, I can add the sample(s) back.